libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -1376,8 +1376,8 @@ string ABIFunctions::calldataAccessFunction(Type const& _type)
 	return createFunction(functionName, [&]() {
 		if (_type.isDynamicallyEncoded())
 		{
-			unsigned int baseEncodedSize = _type.calldataEncodedSize();
-			solAssert(baseEncodedSize > 1, "");
+			unsigned int tailSize = _type.calldataEncodedTailSize();
+			solAssert(tailSize > 1, "");
```

Comment:
Can't `tailSize` be zero if the static size of array is zero?

Reply:
Static arrays of size zero are invalid (they wouldn't serve any purpose anyways).